includes/classes/class-swsales-license.php

```diff
@@ -0,0 +1,136 @@
+					<a class="button button-hero" href="https://www.paidmembershipspro.com/pricing/?utm_source=plugin&utm_medium=pmpro-license&utm_campaign=pricing&utm_content=view-license-options" target="_blank"><?php echo esc_html( 'View Support License Options', 'paid-memberships-pro' ); ?></a>
+				<?php } else { ?>
+					<a class="button button-primary button-hero" href="https://www.paidmembershipspro.com/membership-account/?redirect_to=%2Fmembership-account%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dmembership-account%26utm_content%3Dview-account" target="_blank"><?php echo esc_html( 'Manage My Account', 'paid-memberships-pro' ); ?></a>
+					<a class="button button-hero" href="https://www.paidmembershipspro.com/membership-account/?redirect_to=%2Fnew-topic%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dsupport%26utm_content%3Dnew-support-ticket" target="_blank"><?php echo esc_html( 'Open Support Ticket', 'paid-memberships-pro' ); ?></a>
+				<?php } ?>
+			</p>
+
+			<hr />
+
+            <div class="clearfix"></div>
+
+            <img class="sws_icon alignright" src="<?php echo esc_url( plugins_url( 'images/Sitewide-Sales_icon.png', SWSALES_BASENAME ) ); ?>" border="0" alt="Sitewide Sales Logo" />
+            <?php
+                $allowed_pmpro_license_strings_html = array (
+                    'a' => array (
+                        'href' => array(),
+                        'target' => array(),
+                        'title' => array(),
+                    ),
+                    'strong' => array(),
+                    'em' => array(),		);
+            ?>
+
+            <?php
+                echo '<p>' . sprintf( wp_kses( __( 'Sitewide Sales is distributed under the <a href="%s" title="GPLv2 license" target="_blank">GPLv2 license</a>. This means, among other things, that you may use the software on this site or any other site free of charge.', 'paid-memberships-pro' ), $allowed_pmpro_license_strings_html ), '#' ) . '</p>';
+            ?>
+
+            <?php
+                echo '<p>' . wp_kses( __( '<strong>Sitewide Sales offers plans for automatic updates and premium support.</strong> These plans include a Plus license key which we recommend for all public websites running Sitewide Sales. A Plus license key allows you to automatically update when a new security, bug fix, or feature enhancement is released.' ), $allowed_pmpro_license_strings_html ) . '</p>';
+            ?>
+
+            <?php
+                echo '<p>' . wp_kses( __( '<strong>Need help?</strong> Your license allows you to open new tickets in our private support area. Purchases are backed by a 30 day, no questions asked refund policy.' ), $allowed_pmpro_license_strings_html ) . '</p>';
+            ?>
+
+            <p><a href="#" target="_blank"><?php echo esc_html( 'View Support License Options &raquo;', 'paid-memberships-pro' ); ?></a></p>
```

Comment:
Should be updated to use sitewide-sales text domain and link to Stranger Studios support, and may want to change var name `$allowed_pmpro_license_strings_html`. Also text domain on 116

Reply:
This has been updated.